src/runtime/math.py

```diff
@@ -215,25 +227,76 @@ def fabs(x: f64) -> f64:
         return -x
     return x
 
+@overload
+def fabs(x: i64) -> i64:
+    """
+    Return the absolute value of `x`.
+    """
+    if x < 0.0:
+        return -x
+    return x
+
+@overload
+def fabs(x: i32) -> i32:
+    """
+    Return the absolute value of `x`.
+    """
+    if x < 0.0:
+        return -x
+    return x
 
-def ldexp(x: f64, i: i32) -> f64:
-    return x * (2**i)
+# pow
+# supported data types: i32, i64, f32, f64
+@overload
+def pow(x: f64, y: f64) -> f64:
+    """
+    Return `x` raised to the power  `y`.
+    """
+    if y < 0:
+        raise ValueError('y should be nonnegative')
```

Comment:
Add a test for this error in https://github.com/lcompilers/lpython/tree/main/tests/errors